### ISSUE TITLE
Email list

### ISSUE THREAD

**Issue by eepeacock** [LouisK130/IFCB-Annotate#79]
We are starting to share the annotation tool with other groups who are using our data base. Every time we add updates to the tool, Joe asks me to make sure no one is using the tool, and then has me make sure I reload the tool to see the updates. This is going to become tricky. But, we do have users put in their email addresses when they open accounts. Could this generate a list serve? 

What is the best way to handle the need to let outside users know that updates are happening etc?

**Comment by LouisK130** [comment]
Frankly, I don't think anything catastrophic will happen if there's somebody using the tool when it goes down for an update. If the timing is lucky, they could very well not even notice. In the worst case, they lose the annotations they've applied but not yet submitted.

That said, if you for some reason want a list of all registered users' emails, that's readily available to you.  Check out the blue "admin" button at the top right of the home page. If you're a superuser, you should be able to view a list of all users, emails included.

**Comment by eepeacock** [comment]
Yes, I can see the list of emails. I could copy the emails one by one  
into a list to send an email....

With an expanding number of users, there is an increased need to  
communicate with all the users both regarding updates/bugs and  
classification and tag list.

Maybe it wasn't our original intent to serve other groups in our  
database, but at the moment that is the way things are going.

It's possibly a good idea to just make these communications part of  
the ifcb google group. But, I'm not sure. Many people in the group are  
not using ifcb-annotate.




Quoting Louis Kilfoyle <notifications@github.com>:
